packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -54,63 +54,180 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 
 ### Resource sets
 
-A Resource Set is simply a collection of resources. The following resources are currently supported:
+A Resource Set is simply a collection of resources. There are two types of resource identifiers, resources can either be identified by Okta Resource Names (ORN) or by a REST URL format.
 
-* All Users
-* All Groups
-* A specific Group
-* All Users within a specific Group
-* All Apps
-* All Apps of the same type
-* A specific App
+#### Okta Resource Name (ORN)
 
-### Identifiers
+<ApiLifecycle access="ea" />
 
-#### Resource identifiers
+Okta Resource Names (ORN) is the primary resource identifier. ORNs uniquely identify Okta resources. Currently ORNs are available in early access.
```

Comment:
typically in API docs we use the ea tag in the sections that are early-access

Reply:
Okta Resource Name - ORN, ORNs - Okta Resource Names